Source/bmalloc/libpas/src/test/JITHeapTests.cpp

```diff
@@ -44,13 +44,13 @@ void testAllocateShrinkAndAllocate(unsigned initialObjectSize,
                                    unsigned expectedAlignmentOfFinalObject)
 {
     for (unsigned i = numInitialObjects; i--;) {
-        void* ptr = jit_heap_try_allocate(initialObjectSize);
+        void* ptr = jit_heap_try_allocate(initialObjectSize, pas_compact_allocation_mode);
```

Comment:
If we remove the `allocation_mode` parameter from `jit_heap_try_allocate`, we won't need to pass `pas_compact_allocation_mode`.